README.md

```diff
@@ -384,7 +403,41 @@ server.listen(8080)
 
 ## Migrating to 2.x from 1.x
 
-The main change is the update to `path-to-regexp@2.0.0`, which has a few breaking changes:
+#### Deprecation of arity based error middleware
+
+In previous versions of `router` and `express`, the arity of a function was used to determine
```

Comment:
I would not refer to some module that is not installed with `npm install router` in the readme here if not 100% necessary,  which it doesn't seem to be. Many folks use this module unrelated to express so my thinking is that unless a change in this module will affect express outside of it's own documentation (it doesn't seem like it does) then there us no need to mention it.

Reply:
Sure, I can remove that.